appium-cookies/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+build
+*.log
+.DS\_Store
```

Comment:
I think that .DS_Store is better added to the global gitignore (not the project one) so that it never gets added to any project. it doesn't hurt to have it here too, but the other way means you never have to do this again.